lib/wanda/messaging/event_registry.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Wanda.Messaging.EventsRegistry do
+  @moduledoc """
+  Cloudevents type registry of the internal representations
+  """
+
+  @execution_completed_event "trento.checks.v1.ExecutionCompleted"
+
+  @spec extract_metadata(any) :: {:ok, String.t(), String.t()} | {:error, any()}
+  def extract_metadata(%Wanda.Execution.Result{execution_id: execution_id}),
+    do: {:ok, @execution_completed_event, execution_id}
+
+  def extract_metadata(_), do: {:error, :unsupported_event}
```

Comment:
Can we have this moved into the Mapper module?

Reply:
Fine by me, it doesn't harm eitherway :smile:

Reply:
Changed here https://github.com/trento-project/wanda/pull/11/commits/402a03b6dc8e4ad95ba02f620bcddb5d0c3e9792